apps/precipGauge/server/main.js

```diff
@@ -384,22 +382,6 @@ const doCurveParams = function () {
         );
     }
 
-    if (matsCollections["variable"].findOne({name: 'variable'}) == undefined) {
-        matsCollections["variable"].insert(
-            {
-                name: 'variable',
-                type: matsTypes.InputTypes.select,
-                options: variables,
-                controlButtonCovered: true,
-                default: variables[0],
-                unique: false,
-                controlButtonVisibility: 'none',
-                displayOrder: 2,
-                displayPriority: 1,
-                displayGroup: 1
-            });
-    }
-
```

Comment:
Just curiosity - were these moved elsewhere? I wasn't finding a place they were refactored to so I'm wondering if they were vestigial code.

Reply:
Well, my reply did not go to the right thread. Sigh. Here it is again in the correct place for posterity:

"No, they shouldn’t have been there in the first place. I added them
previously thinking they’d make an eventual scorecard easier, when in fact
they did the opposite, so I just removed them (those are single-variable
apps). Thanks for reviewing the pull request!"

Reply:
Basically, it was code added specifically for the scorecard which the scorecard did not end up needing.

Reply:
Makes sense. Thanks!